modules/platforms/dotnet/Apache.Ignite.Tests/Sql/SqlTests.cs

```diff
@@ -344,8 +344,7 @@ public void TestInvalidSqlThrowsException()
         [Test]
         public void TestCreateTableExistsThrowsException()
         {
-            // TODO: IGNITE-20388 Fix it
```

Comment:
I see that some mentions of IGNITE-20388 exist in cpp files, are we going to do something with them?

Reply:
yes, I'l handle it separatelly